### ISSUE TITLE
Question: Requesting every second affects performance? 

### ISSUE THREAD

**Issue by Ahmdrza** [matdombrock/JSliveList#1]
Hi, i think requesting data every second can affects performance, what do u think?

**Comment by matdombrock** [comment]
Absolutely, this was more of a proof of concept than something that was intended for production so it was never optimized. 

Even as an example though, it would be more useful it it was optimized. 

I'm not sure what would be the best way to go about fixing that though. 

Setting a callback to send data when the client updates something would work in one direction, but the client also needs to update when another client makes a change no matter what. 

I think maybe a socket might work but I'm not sure. I've never worked with web sockets honestly. 

If it turns out that sockets are the way to go, I think it would really make more sense to just create a new project all together since that would probably significantly change the foundation of the code. 

If you can think of a less intensive solution I'm all ears.

**Comment by Ahmdrza** [comment]
I worked on a project last year that had real time factor in it. I also used this technique, I couldn't find any other technique. I wasn't able to use sockets :(
